### DISCUSSION TITLE
Ibis – Ibis project 2024 roadmap

### DISCUSSION THREAD

**Discussion by giscus** [ibis-project/ibis#8493] [Q&A]
# Ibis – Ibis project 2024 roadmap

the portable Python dataframe library

https://ibis-project.org/posts/roadmap-2024-h1/

**Comment by tswast** [comment]
Wonderful working with you all and I'm excited to see how the future unfolds with Ibis.

P.S. It was fun designing the logo. I'm looking forward to seeing the upgraded design when it comes.
